docs/source/installation/mac.rst

```diff
@@ -1,4 +1,9 @@
 Mac
 ===
 
-A stub for mac installation
+The simplest way to install the system dependencies on Mac OS X is with Homebrew.
+
+1. Installing Cairo: ``brew install cairo``
+2. Installing Sox: ``brew install sox``
+3. Installing ffmpeg: ``brew install ffmpeg``
+4. Installing latex (MiKTeX): https://miktex.org/howto/install-miktex-mac
```

Comment:
Did you mean to add `pip install manimlib` (or maybe `pip install --user manimlib`) to the end?

Reply:
Does mac python install includes pip in the PATH? I had to add ``script/`` to PATH under windows.

Reply:
I honestly don't know, as I don't use a mac.